src/components/Card/Card.scss

```diff
@@ -0,0 +1,14 @@
+@import "styles/colors.scss";
+
+.card {
+  color: $color-off-white;
+  font-family: "Open sans";
+
+  position: relative;
```

Comment:
Should be done with tailwind, missed this on my first look.

Reply:
resolved

Reply:
my b, resolved